### PR TITLE
Hide order column for user account

### PR DESCRIPTION
**Pull request by Pearl1594**
Disables the order column in Templates List View for normal user - Addresses issue: https://github.com/apache/cloudstack-primate/issues/473

### CODE REVIEW

**Comment by Pearl1594** [comment]
@blueorangutan package

**Comment by Pearl1594** [comment]
@blueorangutan package

**Comment by rhtyd** [comment]
@blueorangutan package

File: src/views/AutogenView.vue [hunk 1]
```diff
@@ -485,6 +485,10 @@ export default {
         }
       }
 
+      if (!['Admin'].includes(this.$store.getters.userInfo.roletype) && this.apiName === 'listTemplates' && this.selectedFilter !== 'self') {
```
Comment (rhtyd) [inline src/views/AutogenView.vue:488]: @Pearl1594 do we need to check for the selected filter; shouldn't we not display the order column irrespective of that? Maybe check against old UI?

Comment (Pearl1594) [reply src/views/AutogenView.vue:488]: @rhtyd In the old UI - the order column is shown for all users , but the backend API prevents it from ordering them with the following exception: 
```
Domain Admin and regular users can modify only their own Public templates
```
Hence, to adhere to this - I've exposed the order column only for those that it owns

Comment (rhtyd) [reply src/views/AutogenView.vue:488]: @Pearl1594 so if an account owns templates, could they update and change order for their own (public) templates? Instead, we can just enforce that the order columns are shown only to the root admin.

Comment (Pearl1594) [reply src/views/AutogenView.vue:488]: The API allows users and dom admins to change the order of their own public templates. However, I can restrict this only to Root admin

**Comment by davidjumani** [comment]
@blueorangutan package

**Comment by rhtyd** [comment]
@blueorangutan package

**Comment by davidjumani** [comment]
@blueorangutan package

**Comment by Pearl1594** [comment]
@blueorangutan package

**Comment by Pearl1594** [comment]
Yup, after the api is executed, the order gets reset

**Pull request merged**